### PR TITLE
Add templates for the Adversarial QA dataset

### PR DESCRIPTION
**Pull request by abheesht17**

### CODE REVIEW

File: templates/adversarial_qa/adversarialQA/templates.yaml [hunk 1]
```diff
@@ -1,8 +1,159 @@
...
+
+      {{question}}'
+    name: adversarial_qa_4
+    reference: 'Input: Context, Output: Question (generate a question)'
+    task_template: false
+  0e13b79c-850f-45e4-b0b5-f0dc352337c9: !Template
+    id: 0e13b79c-850f-45e4-b0b5-f0dc352337c9
+    jinja: "I know that the answer to \"{{question}}\" is in \"{{context}}\" somewhere.\
+      \ Can you return the offsets of the answer? \n\nSide-note for indexing/exclusive/inclusive\
+      \ range: You know what computer scientists like *wink* \n|||\n{{(answers.answer_start[0],\
```
Comment (craffel) [inline templates/adversarial_qa/adversarialQA/templates.yaml:18]: I'm not sure what you mean by the computer scientists comment here - can you be more specific?

**Review by craffel — CHANGES_REQUESTED** [review]
Thanks! Do you mind adding one template which is just

{{context}}

{{question}}

|||

{{answer}}

?

File: templates/adversarial_qa/adversarialQA/templates.yaml (hunk 1 above)
Comment (abheesht17) [reply templates/adversarial_qa/adversarialQA/templates.yaml:18]: Aah, I just meant 0-indexing, and [inclusive, exclusive) range. I'll change this if it's is on the vague side.

**Comment by abheesht17** [comment]
@craffel, I've done 

```{{context}}

{{question}}

|||

{{answer}}

?
``` 

in ```adversarial_qa_3```.

**Comment by abheesht17** [comment]
@craffel, have a look. I've made the requested change.

File: templates/adversarial_qa/adversarialQA/templates.yaml [hunk 2]
```diff
@@ -1,8 +1,159 @@
...
+    id: 4aed7f08-2285-4e9c-80f6-4c52dd1e760c
+    jinja: 'I want to make a question paper for students. I''ve crawled paragraphs
+      from Wikipedia and now want to test their reading comprehension prowess. I am
+      lazy; can you output a question-answer pair for the paragraph "{{context}}"?
+      |||
+
+      Question: "{{question}}"
+
+
+      Answer:  "{{answers.text[0]}}'
```
Comment (craffel) [inline templates/adversarial_qa/adversarialQA/templates.yaml:60]: You're missing a closing " here.

Comment (abheesht17) [reply templates/adversarial_qa/adversarialQA/templates.yaml]: Fixed.

**Comment by craffel** [comment]
Thanks!

**Pull request merged**